### PR TITLE
Minor doc type migrations tweaks for apps migration

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dannyroberts** [comment]
@nickpell cc: @kaapstorm

File: corehq/doctypemigrations/continuous_migrate.py [hunk 1]
```diff
@@ -25,6 +26,7 @@ def copy_docs(source_db, target_db, doc_ids):
         return
     with IterDB(target_db, new_edits=False) as iter_db:
         for doc in iter_docs(source_db, doc_ids, attachments=True):
+            doc = _insert_attachments(source_db, doc)
```
Comment (kaapstorm) [inline corehq/doctypemigrations/continuous_migrate.py]: @dannyroberts I'm not sure why this works. 

`iter_docs` uses dimagi-utils and POSTs a query to "_all_docs" with "attachments=True" as a query param. `_insert_attachments` uses couchdbkit, which uses restkit, which sends a GET request with "attachments=True" as a query param. 

Why does Couch return the attachments the second time but not the first time? ... And if it's that Couch doesn't include attachments in bulk/list queries, is there a reason we call `iter_docs` with the "attachments=True" param?

Comment (dannyroberts) [inline corehq/doctypemigrations/continuous_migrate.py]: Yeah it's legit confusing. You're right that I could remove `attachments=True` from `iter_docs`. Overall, what we're seeing here is a discrepancy between CouchDB and Cloudant. Cloudant doesn't respect `attachments=true` in views or `_all_docs`, whereas CouchDB does. There's a comment to this effect in the other place where we had to do this, I just was lazy here.

Comment (dannyroberts) [inline corehq/doctypemigrations/continuous_migrate.py]: https://github.com/dimagi/commcare-hq/pull/10578

Comment (kaapstorm) [inline corehq/doctypemigrations/continuous_migrate.py]: :+1:

**Pull request merged**